### PR TITLE
Pep8 updates

### PR DESCRIPTION
**Pull request by wasade**
This is blocking other pending PRs. Should resolve outstanding pep8 issues.

### CODE REVIEW

**Comment by josenavas** [comment]
:+1:

File: biom/util.py [hunk 1]
```diff
@@ -40,6 +40,8 @@
 __url__ = "http://biom-format.org"
 __maintainer__ = "Daniel McDonald"
 __email__ = "daniel.mcdonald@colorado.edu"
+__format_version__ = (2, 1)
```
Comment (ElDeveloper) [inline biom/util.py]: Why were these two things moved here?

**Comment by ElDeveloper** [comment]
*nooooooooo* ... oh well :L

**Comment by josenavas** [comment]
@ElDeveloper why don't you gitter?

**Comment by wasade** [comment]
@ElDeveloper, its the classic chicken and pep8 issue, where the module
level imports need to be the first thing. Normally this is fine, except
that biom.table and biom.parse depends on the versions, so the module level
import of those results in a cyclic hell. This was the only solution I
could find that would preserve API compatibility (for instance qiime's
print_qiime_config.py does a `from biom import __version__`)

**Comment by ElDeveloper** [comment]
OK, that was exactly my concern. So then it should work just fine right?
i.e. projects that do `from biom import __version__` will continue to
work without a problem?

**Comment by wasade** [comment]
That is the intent

**Comment by ElDeveloper** [comment]
:boom: *slow :clap:*

**Pull request merged**